.github/pull_request_template.md

```diff
@@ -1,4 +1,10 @@
-Goals:
+Thanks for your pull request! :rocket:
+
+We know that dev life is hectic, but **please provide a (brief) description** of what your PR does, and how it does it. **Otherwise, your PR cannot be reviewed!**
+
+This policy was agreed upon in a past company retro, and makes everyone's life a little easier. Thanks for your collaboration!
+
```

Comment:
Let's put this part in the comments.  Or else, we probably will see these text in every PR unless the developers delete them manually. 

```suggestion
<!---Decomposing the complex issue into subtasks can help you build it step-by-step. Thanks for your pull request! :rocket: --->
<!---We know that dev life is hectic, but **please provide a (brief) description** of what your PR does, and how it does it. **Otherwise, your PR cannot be reviewed!** --->
<!---This policy was agreed upon in a past company retro, and makes everyone's life a little easier. Thanks for your collaboration!--->
```